README.md

```diff
@@ -21,10 +21,131 @@ It implements the following architecture:
+          
+              `GLFW3_OPENGL_DIR`: '<path to codes workspace>/glfw/glfw-3.2.1/glfw-3.2.1/build/install/lib' (you should be able to find the glfw.lib file)
+              `YARP_USEGLFW3`: check the box
+
+      - Add the following varibales to the Rapid Evironment Edittor:
+             
+              `GLFW3_DIR`= '<path to codes workspace>\glfw\glfw-3.2.1\glfw-3.2.1\build\install' (variable type is String)
+             
+      - Don't forget to configure, generate the cmake (cmake gui --> robotology yarp) and build release mode the yarp using vs15.  
+             
+      - if you have problems with compiling GLFW, [this link](https://www.glfw.org/docs/latest/compile_guide.html#compile_generate) or [here](https://github.com/nigels-com/glew) may help.
+          
+          
+    4. `LibOVR`: to dowload the SDK, you can go to [this website](), choose "Native Windows", then "Core Package: OCULUS SDK for windows". select the version 1.16.0 to download (you can install the latest one as well 1.32 (the last release) but there are some breaking changes in the APIs in version 1.17 and you will probably need to patch the driver software). To download you can go easily to [link](https://developer.oculus.com/downloads/package/oculus-sdk-for-windows/1.16.0/) to download the sdk version 1.16.0.
+   
+      - Extract and place the package in your workspace (in our case in same path of robotology-superbuild), 1inside that there are two libraries which we need: LibOVR, and LibOVRKernel.
+      
+      - Build (both release and debug) the projects inside the sdk using VS15: 
+        
+        '\LibOVR\Projects\Windows\VS2015\LibOVR.vcxproj'
+        
+        '\LibOVRKernel\Projects\Windows\VS2015\LibOVRKernel.vcxproj '
+      
+      - Add the following paths to the variables of robotology/YARP (because the yarpdev which gets the data from the sdk of the ovr and publishes in yarp framework, has dependencies on this library)
+          
+          
+              `LibOVR_LibOVRKernel_INCLUDE_DIR`:'<path to codes workspace>/OculusSDK/LibOVRKernel/Src'
+              
+              `LibOVR_LibOVRKernel_LIBRARY_DEBUG`:'<path to codes workspace>/OculusSDK/LibOVRKernel/Lib/Windows/x64/Debug/VS2015/LibOVRKernel.lib'
+
+              `LibOVR_LibOVRKernel_LIBRARY_RELEASE`: '<path to codes workspace>/OculusSDK/LibOVRKernel/Lib/Windows/x64/Release/VS2015/LibOVRKernel.lib'
+
+              `LibOVR_LibOVR_Extras_INCLUDE_DIR`: '<path to codes workspace>/OculusSDK/LibOVR/Include/Extras'
+
+              `LibOVR_LibOVR_INCLUDE_DIR:PATH`: '<path to codes workspace>/OculusSDK/LibOVR/Include'
+
+              `LibOVR_LibOVR_LIBRARY_DEBUG`: '<path to codes workspace>/OculusSDK/LibOVR/Lib/Windows/x64/Debug/VS2015/LibOVR.lib'
+
+              `LibOVR_LibOVR_LIBRARY_RELEASE`: '<path to codes workspace>/OculusSDK/LibOVR/Lib/Windows/x64/Release/VS2015/LibOVR.lib'
+
+              `YARP_USE_LOBOVR`: check the box
```

Comment:
you can remove white lines between   `LibOVR_LibOVR_INCLUDE_DIR:PATH: '<path to codes workspace>/OculusSDK/LibOVR/Include'` and   `LibOVR_LibOVR_LIBRARY_DEBUG: '<path to codes ...` (etc)

Reply:
I think its readability is higher with a line space in between!